### ISSUE TITLE
New Voucher from Bulk Generate

### ISSUE THREAD

**Issue by smockensturm** [verbb/gift-voucher#79]
`Uncaught TypeError: Craft.ElementEditor is not a constructor`

<img width="549" alt="Screen Shot 2021-12-07 at 2 45 18 PM" src="https://user-images.githubusercontent.com/6586518/145096137-66011ff8-d582-45e1-af8d-04467876b06e.png">

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/gift-voucher` requirement in `composer.json` to:

```
"require": {
  "verbb/gift-voucher": "dev-craft-3 as 2.6.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.7.0](https://github.com/verbb/gift-voucher/releases/tag/2.7.0)
